cedar-policy-validator/src/human_schema/err.rs

```diff
@@ -353,6 +353,8 @@ pub enum ToJsonSchemaError {
     UnknownTypeName(Node<SmolStr>),
     #[error("Use reserved namespace `__cedar`")]
     UseReservedNamespace(Loc),
+    #[error("Invalid name: `{0}`")]
+    InvalidName(SmolStr),
```

Comment:
Can this come with a `Loc` like all the other errors in this enum?

Reply:
We don't need this error variant. Removed.